processor/src/main/java/com/linecorp/decaton/processor/metrics/Metrics.java

```diff
@@ -261,6 +262,12 @@ public class RetryMetrics extends AbstractMetrics {
                                    .description("The number of tasks failed to enqueue in retry topic")
                                    .tags(availableTags.subscriptionScope())
                                    .register(registry));
+
+        public final DistributionSummary retryCount =
+                meter(() -> DistributionSummary.builder("retry.count")
```

Comment:
If this distribution summary is exposed as Prometheus/OpenTelemetry [Histogram](https://prometheus.io/docs/concepts/metric_types/#histogram) metric, IIUC a time series with `_count` suffix will be created. This results in having `retry_count_count`, which could be redundant?

How about naming it something like `retry.task.retries`?

> ...should have a suffix describing the unit, in plural form.
https://prometheus.io/docs/practices/naming/

Reply:
Thank you. Updated in https://github.com/line/decaton/pull/178/commits/4e704e775401c2d6475b35ff79a2bceb97e79768